examples/run_T1CPMG.py

```diff
@@ -32,190 +32,189 @@
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 1),
+                ("delay", config_dict["deadtime_us"]),
+                ("delay", pad_start),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad_end),
+                ("marker", "echo_label", (config_dict["nEchoes"] - 1)),  # 1 us delay
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 1),
+                ("delay", config_dict["deadtime_us"]),
+                ("delay", pad_start),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad_end),
+                ("jumpto", "echo_label"),  # 1 us delay
+                ("delay", config_dict["repetition_us"]),
+                ("jumpto", "start"),
+            ]
+        )
     if not phase_cycling:
-        SpinCore_pp.load([
-            ('marker','start',nScans),
-            ('phase_reset',1),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,0.0),
-            ('delay',vd_us),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',p90_us,0.0),
-            ('delay',tau_us),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,0.0),
-            ('delay',deadtime_us),
-            ('acquire',acq_time_ms),
-            ('delay',pad),
-            ('marker','echo_label',(nEchoes-1)),
-            ('delay_TTL',deblank_us),
-            ('pulse_TTL',2.0*p90_us,0.0),
-            ('delay',deadtime_us),
-            ('acquire',acq_time_ms),
-            ('delay',pad),
-            ('jumpto','echo_label'),
-            ('delay',repetition_us),
-            ('jumpto','start')
-            ])
+        SpinCore_pp.load(
+            [
+                ("marker", "start", config_dict["nScans"]),
+                ("phase_reset", 1),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 0.0),
+                ("delay", vd_us),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", config_dict["p90_us"], 0.0),
+                ("delay", tau_us),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 0.0),
+                ("delay", config_dict["deadtime_us"]),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad),
+                ("marker", "echo_label", (config_dict["nEchoes"] - 1)),
+                ("delay_TTL", config_dict["deblank_us"]),
+                ("pulse_TTL", 2.0 * config_dict["p90_us"], 0.0),
+                ("delay", config_dict["deadtime_us"]),
+                ("acquire", config_dict["acq_time_ms"]),
+                ("delay", pad),
+                ("jumpto", "echo_label"),
+                ("delay", config_dict["repetition_us"]),
+                ("jumpto", "start"),
+            ]
+        )
```

Comment:
again it's own ppg (no function in spincore_pp/ppg) because it is only used in this one script